specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/CommonDefinitions.json

```diff
@@ -3078,6 +3078,10 @@
             "virtualNetworkSubnetId": {
               "description": "Azure Resource Manager ID of the Virtual network and subnet to be joined by Regional VNET Integration.\nThis must be of the form /subscriptions/{subscriptionName}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworks/{vnetName}/subnets/{subnetName}",
               "type": "string"
+            },
+            "managedEnvironment": {
+              "description": "containerapp environment ID. It is used to indentify customer's kube/centauri environment",
```

Comment:
Azure Resource Manager ID of the customer's selected Managed Environment on which to host this app. This must be of the form /subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.App/managedEnvironments/{managedEnvironmentName}

Reply:
Spoke to Navy. To maintain convention. Lets also call this managedEnvironmentId